tests/lib/account.js

```diff
@@ -164,6 +164,46 @@ define([
           );
       });
 
+      test('#reset password with metricsContext metadata', function () {
+        var user = 'test5' + new Date().getTime();
+        var email = user + '@restmail.net';
+        var passwordForgotToken;
+
+        return respond(client.signUp(email, 'oldpassword'), RequestMocks.signUp)
```

Comment:
You could potentially use the `accountHelper.newVerifiedAccount` function to save on the boilerplate?